README.md

```diff
@@ -1,126 +1,110 @@
 
-```
-NAME                                     READY   STATUS    RESTARTS   AGE
-adservice-76bdd69666-ckc5j               1/1     Running   0          2m58s
-cartservice-66d497c6b7-dp5jr             1/1     Running   0          2m59s
-checkoutservice-666c784bd6-4jd22         1/1     Running   0          3m1s
-currencyservice-5d5d496984-4jmd7         1/1     Running   0          2m59s
-emailservice-667457d9d6-75jcq            1/1     Running   0          3m2s
-frontend-6b8d69b9fb-wjqdg                1/1     Running   0          3m1s
-loadgenerator-665b5cd444-gwqdq           1/1     Running   0          3m
-paymentservice-68596d6dd6-bf6bv          1/1     Running   0          3m
-productcatalogservice-557d474574-888kr   1/1     Running   0          3m
-recommendationservice-69c56b74d4-7z8r5   1/1     Running   0          3m1s
-redis-cart-5f59546cdd-5jnqf              1/1     Running   0          2m58s
-shippingservice-6ccc89f8fd-v686r         1/1     Running   0          2m58s
-```
+   ```sh
+   kubectl get pods
+   ```
 
-7. **Access the web frontend in a browser** using the frontend's `EXTERNAL_IP`.
+   After a few minutes, you should see the Pods in a `Running` state:
 
-```
-kubectl get service frontend-external | awk '{print $4}'
-```
+   ```
+   NAME                                     READY   STATUS    RESTARTS   AGE
+   adservice-76bdd69666-ckc5j               1/1     Running   0          2m58s
+   cartservice-66d497c6b7-dp5jr             1/1     Running   0          2m59s
+   checkoutservice-666c784bd6-4jd22         1/1     Running   0          3m1s
+   currencyservice-5d5d496984-4jmd7         1/1     Running   0          2m59s
+   emailservice-667457d9d6-75jcq            1/1     Running   0          3m2s
+   frontend-6b8d69b9fb-wjqdg                1/1     Running   0          3m1s
+   loadgenerator-665b5cd444-gwqdq           1/1     Running   0          3m
+   paymentservice-68596d6dd6-bf6bv          1/1     Running   0          3m
+   productcatalogservice-557d474574-888kr   1/1     Running   0          3m
+   recommendationservice-69c56b74d4-7z8r5   1/1     Running   0          3m1s
+   redis-cart-5f59546cdd-5jnqf              1/1     Running   0          2m58s
+   shippingservice-6ccc89f8fd-v686r         1/1     Running   0          2m58s
+   ```
 
-*Example output - do not copy*
+7. Access the web frontend in a browser using the frontend's external IP.
 
-```
-EXTERNAL-IP
-<your-ip>
-```
+   ```
+   kubectl get service frontend-external | awk '{print $4}'
+   ```
 
-**Note**- you may see `<pending>` while GCP provisions the load balancer. If this happens, wait a few minutes and re-run the command.
+   Visit `https://EXTERNAL_IP` to access your instance of Online Boutique.
 
-8. [Optional] **Clean up**:
+8. Once you are done with it, delete the GKE cluster.
 
-```
-gcloud container clusters delete onlineboutique \
-    --project=${PROJECT_ID} --zone=${ZONE}
-```
+   ```
+   gcloud container clusters delete online-boutique
```

Comment:
**Issue:**
I had a GKE standard cluster (`us-central1-a`) called  `online-boutique` in a different project, and this `delete` command first attempted to delete that cluster instead.

So let's specify the `--region`.
```suggestion
   gcloud container clusters delete online-boutique --region=us-central1
```

It's strange because I even re-ran `gcloud config set project MY_PROJECT_ID` before running the `delete` command `gcloud config set compute/region us-central1`.
See [my logs here](https://screenshot.googleplex.com/44pffbbNjNyc4cu.png).

Reply:
Fixed!